bowtie/templates/report.html.j2

```diff
@@ -3,424 +3,437 @@
-            <li class="nav-item">
-              <a class="nav-link" href="#run-info">Run Info</a>
-            </li>
-            <li class="nav-item">
-              <a class="nav-link" href="#summary">Summary</a>
-            </li>
-            <li class="nav-item">
-              <a class="nav-link" href="#cases">Details</a>
-            </li>
-            {% if generate_dialect_navigation %}
-              <li class="nav-item">
-                <a class="nav-link" href="#"><b>Dialects:</b></a>
-              </li>
-              <li class="nav-item">
-                <a class="nav-link" href="draft2020-12.html">2020-12</a>
-              </li>
-              <li class="nav-item">
-                <a class="nav-link" href="draft2019-09.html">2019-09</a>
-              </li>
-              <li class="nav-item">
-                <a class="nav-link" href="draft7.html">7</a>
-              </li>
-              <li class="nav-item">
-                <a class="nav-link" href="draft6.html">6</a>
-              </li>
-              <li class="nav-item">
-                <a class="nav-link" href="draft4.html">4</a>
-              </li>
-              <li class="nav-item">
-                <a class="nav-link" href="draft3.html">3</a>
-              </li>
-            {% endif %}
-          </ul>
-        </div>
+<head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <title>Bowtie</title>
+  <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.0-alpha3/dist/css/bootstrap.min.css" rel="stylesheet">
+</head>
+
+<body>
+  <nav class="navbar navbar-expand-lg sticky-top bg-light mb-4">
+    <div class="container-fluid">
+      <a class="navbar-brand mb-0 h1" href="#">Bowtie</a>
+      <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarNav"
+        aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
+        <span class="navbar-toggler-icon"></span>
+      </button>
+      <div class="collapse navbar-collapse" id="navbarNav">
+        <ul class="navbar-nav me-auto mb-2 mb-lg-0">
+          <li class="nav-item">
+            <a class="nav-link" href="#run-info">Run Info</a>
+          </li>
+          <li class="nav-item">
+            <a class="nav-link" href="#summary">Summary</a>
+          </li>
+          <li class="nav-item">
+            <a class="nav-link" href="#cases">Details</a>
+          </li>
+          {# If generate_dialect_navigation exists, then render the dropdown navigation  #}
+          {% if generate_dialect_navigation %}
+          <li class="nav-item dropdown">
+            <a class="nav-link dropdown-toggle" href="#" role="button" data-bs-toggle="dropdown" aria-expanded="false">
+              Dialects
+            </a>
+            <ul class="dropdown-menu">
+              <li><a class="dropdown-item" href="draft2020-12.html">2020-12</a></li>
```

Comment:
Also while we're here fixing things, let's maybe use URLs without `.html` at the end, i.e. `href="draft7"` rather than `draft7.html`, etc.

Reply:
Finally, Should I make another commit 
- For adding crossorigin, integrity back again.
- What about keeping the same CDN?
- Or switch to the v5.2.0 and wait until v.5.3.0 get stable then start working on light mode and dark mode themes for the site?

Reply:
If it's not necessary for this PR keep the version exactly as-is without changing and you can bump it in a PR for dark mode of course!